src/featbin/wav-copy.cc

```diff
@@ -26,35 +27,69 @@ int main(int argc, char *argv[]) {
   try {
     using namespace kaldi;
     const char *usage =
-        "Copy archives of wave files\n"
+        "Copy wave file or archives of wave files\n"
         "\n"
-        "Usage:  wav-copy [options...] <wav-rspecifier> <wav-rspecifier>\n"
+        "Usage: wav-copy [options] <wav-rspecifier> <wav-wspecifier>\n"
+        "  or:  wav-copy [options] <wav-rxfilename> [<wav-wxfilename>]\n"
         "e.g. wav-copy scp:wav.scp ark:-\n"
+        "     wav-copy wav.ark:123456 \n"
         "See also: wav-to-duration extract-segments\n";
-    
+
     ParseOptions po(usage);
 
     po.Read(argc, argv);
 
-    if (po.NumArgs() != 2) {
+    if (po.NumArgs() < 1 || po.NumArgs() > 2) {
```

Comment:
please don't make the 2nd arg optional- for Kaldi programs it's not traditional to have optional args like this, so it could be confusing.